docs/designs/table-of-contents.md

```diff
@@ -2,10 +2,10 @@
 
 ## Introduction
 
-Table of contents file which is also called TOC file is used to organize file structure and to provide navigation in the published page. TOC Files must have file name toc.md or toc.yml, notice that file name is case-insensitive.
+Table of contents file which is also called TOC file is used to organize file structure and to provide navigation in the published page. TOC Files must have file name toc.md, toc.yml or toc.json, notice that file name is case-sensitive(depends on platform file-system).
```

Comment:
>is [](start = 218, length = 2)

could be case-sensitive or -nonsensitive depending on the platform

Reply:
using yufei's suggestion